yt_dlp/extractor/picarto.py

```diff
@@ -84,7 +85,7 @@ def _real_extract(self, url):
 
 
 class PicartoVodIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www.)?picarto\.tv/videopopout/(?P<id>[^/?#&]+)'
+    _VALID_URL = r'https?://(?:www.)?picarto\.tv/(videopopout|[a-zA-Z0-9]+/videos)/(?P<id>[^/?#&]+)'
     _TESTS = [{
         'url': 'https://picarto.tv/videopopout/ArtofZod_2017.12.12.00.13.23.flv',
```

Comment:
does this test pass?

if not, add `skip` to it and make your new `only_matching` test into a proper test

Reply:
I did have a question about these tests, since picarto deletes VODs after some time the new test would eventually become invalidated, would that still be ok?

Reply:
that's OK. a download test that will expire is better than no test

Reply:
Sounds good, updated the tests and added the run result in the description.